docs/porting-guides/6.x.x.md

```diff
@@ -156,6 +156,13 @@ snmp_settings:
       ipv4_acl: IPV4
 ```
 
+### Default value of `underlay_ospf_maximum_paths` set to 128
+
+In AVD 5.x, the parameter `underlay_ospf_maximum_paths`(maximum number of OSPF next-hops in an ECMP route) had no default value.
+
+Starting AVD 6.0.0, default value of `underlay_ospf_maximum_paths` is set to `128`.
+Since `128` is already the EOS system default, it will not be rendered in either the structured configuration or the CLI configuration.
```

Comment:
```suggestion
Since `128` is already the EOS system default, it will neither be rendered in the structured configuration nor the CLI configuration.
```